src/site/shared/containers/HistoryBox/index.scss

```diff
@@ -121,4 +142,33 @@ $historybox-width: 250px;
         }
     }
 
+    &__dashedgroupentry {
```

Comment:
this and `&__groupentry` are nearly identical besides a few differences, what you can do is make a class `&__groupentry--dashed` and then below when you apply the className, you can have `historybox__groupentry ${isRedo ? "historybox__groupentry--dashed" : ""}` which applies both classes, it's a little verbose but is the "preferred" way of doing things; but you definitely want to avoid duplication as much as possible

Reply:
Also same thing with `&__entry` and `&__dashedentry`